lms/static/js/discovery/main.js

```diff
@@ -0,0 +1,22 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/discovery/app',
+    'js/discovery/collection',
+    'js/discovery/form',
+    'js/discovery/result_list_view',
+    'js/discovery/filter_bar_view',
+    'js/discovery/search_facets_view'
+], function ($, Backbone, App, Collection, DiscoveryForm, ResultListView, FilterBarView, FacetsBarView) {
```

Comment:
@andy-armstrong Would appreciate your take on @wedaly's concern here. We talked a little yesterday, and we'd like your input if possible. It seems that the right strategy is to assume that things get improved as we integrate RequireJS optimiser.

We have plans for a full performance analysis for all the search-backed features post our current deliverable.